### DISCUSSION TITLE
Acceder a teams del backend

### DISCUSSION THREAD

**Discussion by DaveOval** [Launch-X-Latam/MisionFrontEnd#488] [Team Posts]
Banda.
 para acceder al teams denle en esta parte para que aparezca el front y el back [ara que de ahu puedan acceder 

![image](https://user-images.githubusercontent.com/112228738/200439929-f9bafc52-d38e-49a1-95db-72d68426a1d8.png)

**Comment by Andersonm22** [comment]
![image](https://user-images.githubusercontent.com/83189842/200446257-2ba60bd1-d62c-4dd6-a422-0ff6ff361c6c.png)
 Eso me sale a mi :c

**Comment by AleBT75** [comment]
Eso me salía cuando no estaba con el correo correspondiente, o sea el que me dieron para Launch X, si no es eso habla con soporte por si las dudas

**Comment by Andersonm22** [comment]
> Eso me salía cuando no estaba con el correo correspondiente, o sea el que me dieron para Launch X, si no es eso habla con soporte por si las dudas

tankiu, no me habia dado cuenta de eso, ya lo arregle
